hashes_and_arrays.rb

```diff
@@ -0,0 +1,64 @@
+  # frozen_string_literal: true
+class Library
+  book1 = {
```

Comment:
I think that the main aim of this task is to practice with OOP
It would be great if you use class construction
https://www.tutorialspoint.com/ruby/ruby_classes.htm

Reply:
I'm not sure if I completely understand in what ways I could implement OOP in this task. Could you please, give me more details or a hint?

Reply:
I mean to use accessor, in that way we have opportunity to add new books, make searching etc.
Maybe it is not in scope of this task:)